Dockerfile

```diff
@@ -13,6 +13,10 @@ RUN wget ${ARACHNI_DISTRIBUTION} -P /sectools --output-document arachni.tar.gz &
     mv arachni-${ARACHNI_LONG_VERSION} arachni && \
     rm arachni.tar.gz
 
+# healthchecks: requires curl (if not present in container)
+# RUN apk --update --no-cache add curl
+HEALTHCHECK --interval=3s --timeout=3s --start-period=5s --retries=3 CMD curl --fail http://localhost:8080/status || exit 1
```

Comment:
(see health check pull request from other scanners)

Reply:
since today's commit all scanners have the same timings